### PR TITLE
 automate the use of x86 emulator on apple silicon for dtcw

### PR DESCRIPTION
**Pull request by gusoer**
### All Submissions:

* [X ] Did you update the `changelog.adoc`?
* [ ] Does your PR affect the documentation?
* [ ] If yes, did you update the documentation or create an issue for updating it?

The source of the documentation can be found in `/src/docs/`.

If you didn't find the time to update docs, please create an issue as reminder to do so.

### Your first submission

* [ X] Welcome to the list of contributors! If you have any questions, feel free to ask them by creating a new issue
* [ X] Have you added your name to the list of [30_community.adoc](https://github.com/docToolchain/docToolchain/blob/ng/src/docs/10_about/30_community.adoc)?

inspiration: https://github.com/stevemao/github-issue-templates

### CODE REVIEW

File: dtcw [hunk 1]
```diff
@@ -417,6 +408,11 @@ if [ "${doJavaCheck}" = true ] ; then
     check_java
 fi
 
-# echo "Command to invoke: ${command}"
+# use x86 emulator on Apple M1/M2
+emu=`[ $os = "Darwin" ] && [ $arch = "arm64" ] && echo "arch -x86_64"`
```
Comment (rdmueller) [inline dtcw]: can we formulate this a bit more readable?

```
emu=""
if $os = "Darwin" && $arch = "arm64" ; then
   emu="arch -x86_64"
fi
```

(don't know if this is the correct syntax)

this is longer but I guess it helps to maintain the source

**Comment by rdmueller** [comment]
thanx for this great submission. What do you think about my comment?

File: dtcw (hunk 1 above)
Comment (gusoer) [reply dtcw]: sure, fine for me - I'll change it a bit to make it more readable

Comment (gusoer) [reply dtcw]: ... done ;-)

**Review by rdmueller — APPROVED** [review]

**Comment by rdmueller** [comment]
Welcome to the list of Contributors!
And thanx for the PR!

**Pull request merged**